exporter/otlp/test/opentelemetry/exporter/otlp/exporter_test.rb

```diff
@@ -182,6 +182,14 @@
       _(result).must_equal(FAILURE)
     end
 
+    it 'returns FAILURE when encryption to receiver endpoint fails' do
+      stub_request(:post, 'https://localhost:4317/v1/traces').to_raise(OpenSSL::SSL::SSLError.new('enigma wedged'))
+      span_data = create_span_data
+      exporter.stub(:backoff?, ->(**_) { false }) do
```

Comment:
Updated to use arcane stabby lambda to short-circuit backoff and retries. I'm open to making this more verbose and maybe a local variable to give names to what's going on here to reduce the eldritch Ruby..

Reply:
I think this is fine.